### PR TITLE
Hide hidden values when service-apply called.

### PR DESCRIPTION
**Pull request by ahmdrz**
I'm not sure about what I wrote :speak_no_evil: because I didn't test it, But I want this option for `CI/CD` to hide hidden values when calling service-apply.

### CODE REVIEW

**Comment by ahmdrz** [comment]
Sorry about other changes. `autopep8` changed them.

**Comment by Anonycoders** [comment]
@ahmdrz dear ahmadreza can you please provide more info about what you mean about values?
what values do you mean?
ENV variables or what?

**Comment by ahmdrz** [comment]
Yes, ENV variables. I set a secret variable in `gitlab` and I used it as a parameter in service apply.
So in all of them, the variable is hidden but when `fandogh service apply` called, it will appear in Gitlab CI logs.

**Comment by s-soroosh** [comment]
@ahmdrz Thanks. 
we will review this PR ASAP.

File: fandogh_cli/service_commands.py [hunk 1]
```diff
@@ -171,7 +176,9 @@ def service_details(service_name):
 @click.option('-p', '--parameter', 'parameters', help='Manifest parameters', multiple=True)
 @click.option('-d', 'detach', is_flag=True, default=False,
               help='detach terminal.')
-def service_apply(file, parameters, detach):
+@click.option('-h', '--hide', is_flag=True, default=False,
```
Comment (s-soroosh) [inline fandogh_cli/service_commands.py]:
```suggestion
@click.option('-h', '--hide', 'hide_manifest', is_flag=True, default=False,
```

**Comment by ahmdrz** [comment]
This update will help to make ENV variables privately on projects on Gitlab that has some members with `guest` or `reporter` permission. I need this feature :smile:

**Comment by s-soroosh** [comment]
Hi @ahmdrz,  
We have to test it  against staging to make sure it doesn't have any negative impact on customers. We do finish the tests Today and merge it. 

Thanks

**Comment by ahmdrz** [comment]
Good to hear. Thank you @psycho-ir and Fandogh team.

**Pull request merged**